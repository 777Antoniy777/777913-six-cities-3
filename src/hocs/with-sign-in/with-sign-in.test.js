import React from 'react';
import renderer from "react-test-renderer";
import withSignIn from "./with-sign-in";

// set mocha data
const props = [];

const MockComponent = () => {
  return (
    <div></div>
  );
};

const SignInWrappedHoc = withSignIn(MockComponent);

it(`render withSignIn`, () => {
  const tree = renderer.create((
    <SignInWrappedHoc
      props={props}
    />
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
