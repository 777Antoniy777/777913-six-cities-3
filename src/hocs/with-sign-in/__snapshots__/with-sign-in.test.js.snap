// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render withSignIn 1`] = `<div />`;
